cypress/integration/common/report.js

```diff
@@ -169,7 +169,7 @@ Then('each list item links to the post page', () => {
 Then('I can visit the post page', () => {
   cy.contains(annoyingUserWhoMutedModeratorTitle).click()
   cy.location('pathname').should('contain', '/post')
-    .get('title').should('contain', annoyingUserWhoMutedModeratorTitle)
+    .get('.base-card .title').should('contain', annoyingUserWhoMutedModeratorTitle)
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-02-20T22:18:15Z" title="Thursday, February 20th 2020, 11:18:15 pm +01:00">Feb 20, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -169,7 +169,7 @@ Then('each list item links to the post page', () => {
 Then('I can visit the post page', () => {
   cy.contains(annoyingUserWhoMutedModeratorTitle).click()
   cy.location('pathname').should('contain', '/post')
-    .get('h3').should('contain', annoyingUserWhoMutedModeratorTitle)
+    .get('.ds-card-content .hyphenate-text').should('contain', annoyingUserWhoMutedModeratorTitle)
```


Code creep?

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-02-20T22:51:58Z" title="Thursday, February 20th 2020, 11:51:58 pm +01:00">Feb 20, 2020</time>_

---


Could be, yes.
It failed locally, but that was in headless. Also, I was reading cypress docs and they suggest to never use tags as selectors.
We could open another PR to clean up our cypress code though